__tests__/app.test.js

```diff
@@ -133,6 +133,15 @@ describe("GET /api", () => {
                     expect(body.msg).toBe("Bad request");
                 })
             })
+            test("200: Should retrieve an array of all article objects with the necessary properties (now including COMMENT COUNT AS IF WE DIDN'T ALREADY WRITE FOR IT!)", () => {
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.article).toHaveProperty("comment_count", expect.any(Number));
+                    expect(body.article.comment_count).toBe(11);
```

Comment:
I would include the rest of the properties in this too

Reply:
This has been refactored to test ALL response properties of a get request to the articles endpoint.